modules/nf-core/isoseq3/cluster/main.nf

```diff
@@ -4,8 +4,8 @@ process ISOSEQ3_CLUSTER {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/isoseq3:3.8.1--h9ee0642_0' :
-        'biocontainers/isoseq3:3.8.1--h9ee0642_0' }"
+        'https://depot.galaxyproject.org/singularity/isoseq3:4.0.0--h9ee0642_0' :
+        'biocontainers/isoseq3:4.0.0--h9ee0642_0' }"
```

Comment:
shouldn't this be isoseq4 then?

Reply:
There is an image with name isoseq (the dropped the 3). 
I can rename the module to `isoseq` if you wish.

Reply:
as you prefer, I'm thinking along the lines of what we did for:
https://github.com/nf-core/modules/tree/master/modules/nf-core/gatk
https://github.com/nf-core/modules/tree/master/modules/nf-core/gatk4
https://github.com/nf-core/modules/tree/master/modules/nf-core/gatk4spark

Reply:
If think you're right and the module should stick to most recent name. 
Is there a risk to rename the module directory (isoseq3 to isoseq)?

Reply:
I updated the name of `isoseq3` modules to `isoseq`.